src/vs/platform/windows/electron-main/window.ts

```diff
@@ -246,6 +247,19 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 				if (!isMacintosh) {
 					options.frame = false;
 				}
+
+				if (isWindows) {
+					// This logic will not perfectly guess the right colors to use on initialization,
+					// but prefer to keep things simple as it is temporary and not noticeable
+					const titleBarColor = this.themeMainService.getWindowSplash()?.colorInfo.titleBarBackground ?? this.themeMainService.getBackgroundColor();
+					const symbolColor = Color.fromHex(titleBarColor).isDarker() ? '#FFFFFF' : '#000000';
+
+					options.titleBarOverlay = {
+						height: 29,
```

Comment:
Where is that `29` number coming from?

Reply:
On Windows 11, we size the title bar at 30 pixels, however 1px is eaten by the native window border, so the smallest height of the title bar is 29px. I had implemented a lot of custom logic here to be completely accurate, but as above, preferring simplicity here